MM-27931, MM-30158 Allow 'multi-selecting' channels in the LHS (#4208)
mattermost/docs: source/help/getting-started/organizing-conversations.rst

**Collapsible custom categories**: Create custom categories in the sidebar to group channels together for easier navigation (e.g. “Design” or “Marketing”). Collapse categories to display only unread channels and reduce unnecessary scrolling.

> **Using the Mattermost Web or Desktop App, select multiple channels at a time. To select sequential channels, press and hold SHIFT. To select non-sequential channels press and hold CMD (for Mac) or CTRL (for Windows/Linux).

Author: ```suggestion
**Using the Mattermost Web or Desktop App, select multiple channels at a time. To select sequential channels, press and hold SHIFT. To select non-sequential channels press and hold CMD (for Mac) or CTRL (for Windows/Linux). To clear channel selections, press ESC.
```

> **Multi-select channels**: Using the Mattermost Web or Desktop App, select multiple channels at a time. Press and hold SHIFT to select sequential channels. Press and hold CMD (for Mac) or CTRL (for Windows/Linux) to select non-sequential channels.

Reviewer: ```suggestion
**Multi-select channels**: Using the Mattermost Web or Desktop App, select multiple channels at a time. To select sequential channels, press and hold SHIFT. To select non-sequential channels press and hold CMD (for Mac) or CTRL (for Windows/Linux).
```

> **Multi-select channels**: Using the Mattermost Web or Desktop App, select multiple channels at a time. To select sequential channels, press and hold SHIFT. To select non-sequential channels press and hold CMD (for Mac) or CTRL (for Windows/Linux).

Author: ```suggestion
**Multi-select channels**: Using Mattermost Cloud, select multiple channels at a time. To select sequential channels, press and hold SHIFT. To select non-sequential channels press and hold CMD (for Mac) or CTRL (for Windows/Linux). The ability to multi-select channels is coming to the Mattermost Web and Desktop Apps in a future release.
```

> **Using the Mattermost Web or Desktop App, select multiple channels at a time. To select sequential channels, press and hold SHIFT. To select non-sequential channels press and hold CMD (for Mac) or CTRL (for Windows/Linux). To clear channel selections, press ESC.

Reviewer 2: 0/5 I think this should fall under the feature of **Drag and drop channels and categories** rather than being placed here

Author: @esethna Once multiple channels are selected, end users can drag and drop those selected channels elsewhere, or they can use the **More options** menu to move those selected channels. As a result, I positioned the selection options ahead of the move options in the product documentation (to match order of operations).

Thoughts?

> **Multi-select channels**: Using Mattermost Cloud, select multiple channels at a time. To select sequential channels, press and hold SHIFT. To select non-sequential channels press and hold CMD (for Mac) or CTRL (for Windows/Linux). The ability to multi-select channels is coming to the Mattermost Web and Desktop Apps in a future release.

Author: ```suggestion
**Using the Mattermost Web or Desktop App, select multiple channels at a time. To select sequential channels, press and hold SHIFT. To select non-sequential channels press and hold CMD (for Mac) or CTRL (for Windows/Linux).
```
